setup.py

```diff
@@ -25,11 +39,18 @@
         "ipython==7.13.0",
         "jsonschema==3.2.0",
         "pyyaml==5.3.1",
-        "pandas==1.0.3",
-        "tqdm==4.46.0"
+        "pandas==1.0.3"
     ],
     package_data={"": ["*.txt"]},
     include_package_data=True,
     scripts=["bin/blabla"],
     zip_safe=False,
+    classifiers=[
+    'Development Status :: 4 - Beta',      # Chose either "3 - Alpha", "4 - Beta" or "5 - Production/Stable" as the current state of your package
+    'Intended Audience :: Developers',      # Define that your audience are developers
```

Comment:
Are developers the intended audience? Recommend `Intended Audience :: Healthcare Industry' and `Intended Audience :: Science/Research' instead.

Reply:
Have replaced this with "Intended Audience :: Healthcare Industry" now